modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -26,7 +26,7 @@ and the `bmc` parameter for the `install-config.yaml` file.
 
 | `bootstrapExternalStaticDNS`
 |
-| The static network DNS of the bootstrap node. This can be useful in environments without a DHCP server.
+| The static network DNS of the bootstrap node. You must set this value when deploying a cluster with static IP addresses when there is no DHCP server on the bare-metal network. If you do not set this value, the installation program will use the value from `bootstrapExternalStaticGateway`, which causes problems when the IP address values of the gateway and DNS are different.
```

Comment:
I don't see "Dynamic Host Configuration Protocol (DHCP)" anywhere on this page so that users know what it is. Maybe add it to the first use?